scanner/scanlog/scanlog.go

```diff
@@ -104,25 +105,26 @@ func dumpData(entry *ct.LogEntry) {
 
 // Prints out a short bit of info about |cert|, found at |index| in the
 // specified log
-func logCertInfo(entry *ct.LogEntry) {
-	if entry.X509Cert != nil {
-		log.Printf("Process cert at index %d: CN: '%s'", entry.Index, entry.X509Cert.Subject.CommonName)
-		dumpData(entry)
+func logCertInfo(entry *ct.RawLogEntry) {
+	parsedEntry, err := entry.ToLogEntry()
+	if err != nil || parsedEntry.X509Cert == nil {
```

Comment:
Note that `err` can be a non-nil non-fatal error here. Might be worth changing the condition to `IsFatal || cert == nil`.